examples/tcx/tcx.c

```diff
@@ -0,0 +1,47 @@
+//go:build ignore
+
+#include "common.h"
+
+char __license[] SEC("license") = "Dual MIT/GPL";
+
+/* Define an ARRAY map for storing ingress packet count */
+struct {
```

Comment:
It's a shame we haven't worked on an API to interact with static variables defined in an ELF. That would've made this example much more elegant.

Reply:
Reaching you out soon on Slack to discuss this :)